src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -263,6 +277,9 @@ describe('RefinementList', () => {
           templates: {
             item: item => item,
             searchableNoResults: x => x,
+            reset: 'reset',
```

Comment:
I thought we decided on `searchableReset` etc. here?

Reply:
It is still called `searchableReset` in the `refinementList` widget, but renamed `reset` [when passed to the component](https://github.com/algolia/instantsearch.js/blob/684cde80f6093c8f10625551c8a4992959711f9f/src/widgets/refinement-list/refinement-list.js#L19-L43). This for the component not to host the logic of the template name (see https://github.com/algolia/instantsearch.js/pull/3489#discussion_r256788379).

Reply:
missed that bit.